SYCL/Matrix/element_wise_all_ops_int8.cpp

```diff
@@ -0,0 +1,251 @@
+//==----------- element_wise_all_ops_int8.cpp  - DPC++ joint_matrix---------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
```

Comment:
AFAIK, we don't need these header in the tests, please remove it.

Reply:
Is this something that has changed recently because I still see it in other tests too?
Please send me the new BKM

Reply:
It looks like some of the tests contain this header, some of them don't. For example, https://github.com/intel/llvm-test-suite/blob/intel/SYCL/Plugin/enqueue-arg-order-buffer.cpp

Since this is inessential and I can't find exact documentation or instruction, I don't want to hold back this PR.